certif/tests/acceptance/session-details-certification-candidates_test.js

```diff
@@ -115,6 +120,24 @@ module('Acceptance | Session Details Certification Candidates', function(hooks)
           assert.contains('Voir le détail');
         });
 
+        test('it should display partially the list of certification candidates', async function(assert) {
```

Comment:
C'est un poil mal formulé, on dirait qu'on affiche la liste partiellement. Je pense qu'il n'y a pas besoin d'être aussi précis. Bientôt ce sera la nouvelle liste officielle avec les colonnes officielles, donc un simple it should display the list of certification candidates me va très bien perso